cmd/keycmd/list.go

```diff
@@ -83,17 +100,61 @@ func printAddresses(keyPaths []string) error {
 			}
 			if !cAdded {
 				strC := sk.C()
-				table.Append([]string{keyName, "C-Chain (Ethereum hex format)", strC, "All"})
+				balanceStr, err := getCChainBalanceStr(ctx, fujiCClient, strC)
+				if err != nil {
+					return err
+				}
+				table.Append([]string{keyName, "C-Chain (Ethereum hex format)", strC, balanceStr, "All"})
 			}
 			cAdded = true
 
 			strP := sk.P()
 			for _, p := range strP {
-				table.Append([]string{keyName, "P-Chain (Bech32 format)", p, net})
+				balanceStr := "0"
+				if net == models.Fuji.String() {
+					var err error
+					balanceStr, err = getPChainBalanceStr(ctx, fujiPClient, p)
+					if err != nil {
+						return err
+					}
+				}
+				table.Append([]string{keyName, "P-Chain (Bech32 format)", p, balanceStr, net})
 			}
 		}
 	}
 
 	table.Render()
 	return nil
 }
+
+func getCChainBalanceStr(ctx context.Context, cClient ethclient.Client, addrStr string) (string, error) {
+	addr := common.HexToAddress(addrStr)
+	ctx, cancel := context.WithTimeout(ctx, constants.RequestTimeout)
+	balance, err := cClient.BalanceAt(ctx, addr, nil)
+	cancel()
+	if err != nil {
+		return "", err
+	}
+	if balance.Cmp(big.NewInt(0)) == 0 {
+		return "0", nil
+	}
+	balance = balance.Div(balance, big.NewInt(int64(units.Avax)))
```

Comment:
Nit: Could do this just in one calculation (possibly just uint64)

Reply:
I'm not sure to understand the suggestion. Balances got from c-chain have an extra units.Avax multiplier. I split the division into this first extra c-chain multiplier part (to obtain nAvax), ignoring the decimal part.
Afterthat I convert nAvax to avax, keeping record of the multiplier.

Reply:
It's really just a nit. We could just divide only once instead of twice. On the other hand, this makes it clearer to understand why. So up to you,